src/composables/accounts.ts

```diff
@@ -47,6 +48,25 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
     return accountsAddressList.value.includes(address);
   }
 
+  /**
+   * @param {string} address - the address to format.
+   * @returns {string} - returns the account's name if it exists,
+   * otherwise it returns the translated string "Account" + the account's index +1.
+  */
+  function getAccountNameByAddress(address: string): string {
+    const { idx, name } = getAccountByAddress(address) || {};
+
+    if (name) {
+      return name;
+    }
+
+    if (idx) {
+      return `${i18n.t('pages.account.heading')} ${idx + 1}`;
```

Comment:
This field was removed in #2036. It is now in `common`.

Reply:
didn't find any changes to the pages account heading in #2036  and the translations still there https://github.com/aeternity/superhero-wallet/blob/develop/src/popup/locales/en.json#L500